build.gradle

```diff
@@ -68,7 +68,7 @@ dependencies {
     "io.kamon:kamon-core_%%:${kamonVersion}",
     "com.typesafe.akka:akka-actor_%%:${akkaVersion}",
     "com.typesafe.akka:akka-testkit_%%:${akkaVersion}",
-    "io.kamon:kamon-prometheus_%%:1.0.0",
+    "io.kamon:kamon-prometheus_%%:2.0.0-RC1",
```

Comment:
Let's just reference this version from the properties like `"io.kamon:kamon-prometheus_%%:${kamonVersion}"`